tests/baselines/reference/canWatch/getDirectoryToWatchFailedLookupLocationUnc.baseline.md

```diff
@@ -138,7 +138,7 @@ Root: //vda1cs4850/folderAtRoot
 | //vda1cs4850/somefile.d.ts                                                                               |                                                                                                          |           |
 | //vda1cs4850/dir/somefile.d.ts                                                                           |                                                                                                          |           |
 | //vda1cs4850/dir/subdir/somefile.d.ts                                                                    |                                                                                                          |           |

```

Comment:
What does it mean when `Recursive` is blank?

Reply:
If second column (path to watch) is blank recursive is blank meaning these kind of paths will be ignored for setting directory watchers